### PR TITLE
FaF Group name not appearing on the Case created in CM

### PR DESCRIPTION
**Pull request by PiperS52**
## Changes in this PR

- when a FaF request is made with a group (MAT or Fed) this is now displayed in case management

### After
Screenshot 2022-02-22 at 12 15 37

Screenshot 2022-02-24 at 15 13 23

### CODE REVIEW

File: app/services/submit_framework_request.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def user
 
   # @return [Support::Organisation]
```
Comment (kennyevil) [inline app/services/submit_framework_request.rb:48]: Might need to update the documentation here.

**Review by kennyevil — APPROVED** [review]

File: app/services/submit_framework_request.rb [hunk 2]
```diff
@@ -45,9 +45,9 @@ def user
     @user ||= FrameworkRequestPresenter.new(request).user
   end
 
-  # @return [Support::Organisation]
+  # @return [Support::Organisation] or [Support::EstablishmentGroup]
```
Comment (peterdavidhamilton) [inline app/services/submit_framework_request.rb:48]: `[Support::Organisation, Support::EstablishmentGroup]`

Comment (peterdavidhamilton) [reply app/services/submit_framework_request.rb:48]: FYI this is another point where the proposed service function needs to be used to span the divide.

**Review by peterdavidhamilton — APPROVED** [review]
Follow up - it needs to be rolled in to the FaF refactor and could do with a school test for completeness

**Comment by PiperS52** [comment]
[FaF Group name not appearing on the Case created in CM](https://trello.com/c/HGRJcSGw/403-faf-group-name-not-appearing-on-the-case-created-in-cm)

**Pull request merged**